notebook/static/notebook/js/actions.js

```diff
@@ -437,7 +437,18 @@ define(function(require){
                 $('.header-bar').toggle();
                 events.trigger('resize-header.Page');
             }
-        }
+        },
+        'escape': {
```

Comment:
Escape is used excessively in most desktop programs as both "close" and "deselect".  I think the list is actually:

* Close
* Deselect

Where command mode is entered by the action of deselecting codemirror.  I'd disagree that these two actions are too many.